### ISSUE TITLE
Import/Export Wallet File for XMR

### ISSUE THREAD

**Issue by HardenedSteel** [cake-tech/cake_wallet#1338]
Since Monero requires synchronization using multiple wallet app requires multiple synchronization, I want to able export/import my wallet file to avoid that.

**Comment by tuxpizza** [comment]
So if I'm understanding your feature request, you want the ability to export the complete wallet state of a _single_ Monero wallet? You can already backup the sync state for all the wallets in a single large backup but not each of them individually

**Comment by HardenedSteel** [comment]
@tuxpizza: Yes I need for single wallet and backup files only work for cake/monero.com wallet, I cannot use with other wallet apps.
